trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require './nameable'
+class TrimmerDecorator < Nameable
+  def initialize(nameable = Nameable.new)
+    super()
+    @nameable = nameable
+  end
+

```

Comment:
- [ ] I think it would be better if you remove the constructor method(`def initialize(nameable = Nameable.new)`) as the `TrimmerDecorator` has already been inherited from `Nameable` in order to not repeat yourself.